tests/functional/mesh/work_test.go

```diff
@@ -23,9 +23,10 @@ func checkSkipKube(t *testing.T) {
 }
 
 func tearDown(controllers map[string]*receptorcontrol.ReceptorControl, m *mesh.CLIMesh) {
-	defer m.WaitForShutdown()
-	defer m.Destroy()
+	// defer m.WaitForShutdown()
```

Comment:
Why is that all commented out? Could you add a comment describing why this is done?